spec/Candid.md

```diff
@@ -1001,12 +1035,12 @@ The relations above have certain properties. As in the previous section, `<v> :
 
 * Roundtripping:
   ```
-  (<v> : <t>) ⟺ C[<t> <: <t>](<v>) = <v>
+  (<v> : <t>) ⟺ <v> : <t> ~> <v> : <t>
   ```
 
```

Comment:
````suggestion

* Well-typedness
  ```
  <v> : <t> ~> <v'> : <t'> ⇒ <v'> <: <t'>
  ```

````

It remains to be seen if such theorems need `<v> : <t>` as an assumption. A technicality, but still. Maybe we should state explicit that the relation is only defined on well-typed input. Or maybe it holds by construction.